openmc/mgxs/__init__.py

```diff
@@ -7,10 +7,15 @@
 
 GROUP_STRUCTURES = {}
 """Dictionary of commonly used energy group structures, including "CASMO-X" (where X
-is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code.
+is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code and other commonly
+used activation energy group structures "VITAMIN-J-175", "TRIPOLI-315", "CCFE-709" 
+and "UKAEA-1102"
 
 .. _CASMO: https://www.studsvik.com/SharepointFiles/CASMO-5%20Development%20and%20Applications.pdf
-
+.. _activation groups http://www.ccfe.ac.uk/assets/Documents/easy/CCFE-R(11)11.pdf page 170
+.. _activation groups https://fispact.ukaea.uk/wiki/Keyword:GETXS
+.. _CCFE-709 https://fispact.ukaea.uk/wiki/CCFE-709_group_structure 
+.. _UKAEA-1102 https://fispact.ukaea.uk/wiki/UKAEA-1102_group_structure
```

Comment:
You need a colon after the label. Otherwise, there is an error when building documentation:
```
/home/romano/openmc/openmc/mgxs/__init__.py:docstring of openmc.mgxs.GROUP_STRUCTURES:7: WARNING: malformed hyperlink target.
/home/romano/openmc/openmc/mgxs/__init__.py:docstring of openmc.mgxs.GROUP_STRUCTURES:8: WARNING: malformed hyperlink target.
/home/romano/openmc/openmc/mgxs/__init__.py:docstring of openmc.mgxs.GROUP_STRUCTURES:9: WARNING: malformed hyperlink target.
/home/romano/openmc/openmc/mgxs/__init__.py:docstring of openmc.mgxs.GROUP_STRUCTURES:10: WARNING: malformed hyperlink target.
```
More fundamentally, these hyperlinks aren't actually used by the documentation above, meaning they should either be removed or used (you can see in the CASMO case, the link above appears as `CASMO_`)

Reply:
Ah I was wondering how that worked.